api/migrations/0001_initial.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.8 on 2016-10-03 03:16
+from __future__ import unicode_literals
```

Comment:
Since we're using 3.5, this line is unnecessary. Unless 2.7 is fully supported, I think it makes sense to remove partial support like this. And I don't really see any reason to support 2.7.